### PR TITLE
Jetpack Pricing Page: add agencies discount FAQ question

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Adds a new question in the FAQ that exists on the Jetpack Pricing page.

#### Testing instructions

* Fire up this PR.
* Go to Calypso Green and head to `/pricing/`.
* Ensure that you see the new question + answer, the link is working, and that events are working.

#### Screenshot

image

Related to 1199980337313591-as-1201692040304222

### CODE REVIEW

File: client/my-sites/plans/jetpack-plans/faq/index.tsx [hunk 1]
```diff
@@ -63,19 +62,29 @@ const JetpackFAQ: React.FC = () => {
...
+				</FoldableFAQ>
 				<FoldableFAQ
 					id="wpcom-account"
 					question={ translate( 'Why do I need a WordPress.com account?' ) }
 					onToggle={ onFaqToggle }
 				>
 					{ translate(
-						"Many of Jetpack's core features make use of the WordPress.com cloud. In order to make sure everything" +
-							" works correctly, Jetpack requires you to connect a free WordPress.com account. If you don't already" +
+						'Many of Jetpack’s core features make use of the WordPress.com cloud. In order to make sure everything' +
```
Comment (a8ci18n) [inline client/my-sites/plans/jetpack-plans/faq/index.tsx:83]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Many of Jetpack's core features make use of the WordPress.com cloud. In order to make sure everything works correctly, Jetpack requires you to connect a free WordPress.com account. If you don't already have an account you can easily create one during the connection process.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-pricing-page-faq-agencies-discount)

**Review by madebynoam — APPROVED** [review]
Lgtm

**Review by monsieur-z — APPROVED** [review]
Tests as expected 👍

File: client/my-sites/plans/jetpack-plans/faq/index.tsx [hunk 2]
```diff
@@ -63,19 +62,29 @@ const JetpackFAQ: React.FC = () => {
...
 					) }
 				</FoldableFAQ>
-
+				<FoldableFAQ
+					id="volume-discount"
+					question={ translate( 'Do you have any discounts for multiple sites?' ) }
+					onToggle={ onFaqToggle }
+				>
+					{ translate(
+						'Anyone with at least five websites can join our licensing platform and enjoy a 25% discount across all Jetpack products! You can learn more about our {{agenciesLink}}licensing platform and agency program here{{/agenciesLink}}.',
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/faq/index.tsx:71]: @rcanepa, would you know if that 25% percent is stored in a variable in Calypso already?

Comment (rcanepa) [reply client/my-sites/plans/jetpack-plans/faq/index.tsx:71]: Not to my knowledge. I think this is the first mention of this discount on Calypso, but I could be wrong.

Comment (monsieur-z) [reply client/my-sites/plans/jetpack-plans/faq/index.tsx:71]: Thanks for your input Renzo.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7103402

Thank you @keoshi for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7103402).

**Pull request merged**